mlflow/langchain/_langchain_autolog.py

```diff
@@ -248,7 +272,7 @@ def __init__(self, *args, **kwargs):
         artifacts_dir=f"artifacts-{session_id}-{inference_id}",
         tags=tags,
     )
-    args, kwargs = _inject_mlflow_callback(func_name, mlflow_callback, args, kwargs)
+    args, kwargs = _inject_mlflow_callbacks(func_name, [mlflow_callback], args, kwargs)
```

Comment:
Does it have to be a list? We only use single callback both before/after trace anyway.

Reply:
It doesn't have to be, actually I make this change to avoid the conflicts with tracing branch https://github.com/mlflow/mlflow/blob/c04b120aa65c01a04de7078874c8bc1edff9dbaa/mlflow/langchain/_langchain_autolog.py#L98 But I think it's fine as list here makes things easier, and if we want to add more callbacks in the future